lib/Arch/AArch64/Runtime/Instructions.cpp

```diff
@@ -29,6 +29,8 @@
 
 // clang-format on
 
+State __remill_state;
```

Comment:
Add a comment describing the purpose of this, that is roughly like your comment in the PR itself where you show what the IR looks like.

Reply:
I suppose this will show up for every architecture, so maybe it can be short and sweet, and then reference your actual PR comment by link :-D

Reply:
Does this need to be in an `extern "C" {` block to get C mangling?

Reply:
Added comments in [5b78167](https://github.com/lifting-bits/remill/pull/631/commits/5b78167c33ae212b4bc4ad0ce8b438febc05de0b).

As for the `extern "C"`, the declaration in `Instruction.cpp` isn't within one of those blocks so I don't think the definition needs to be either. And we seem to be able to get the type fine.